kubernetes/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: prn-maps-api
+  labels:
+    app: prn-maps-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prn-maps-api
+  template:
+    metadata:
+      labels:
+        app: prn-maps-api
+    spec:
+      containers:
+        - name: prn-maps-api
+          image: zooniverse/prn-maps-api:latest
+          ports:
+            - containerPort: 80
+          env:
+            - name: RACK_ENV
+              value: "production"
+            - name: AWS_DEFAULT_REGION
+              value: "us-east-1"
+            - name: ROLLBAR_ACCESS_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: prn-maps-api
+                  key: rollbar_token
+            - name: BASIC_AUTH_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: prn-maps-api
+                  key: basic_auth_username
+            - name: BASIC_AUTH_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: prn-maps-api
+                  key: basic_auth_password
+            - name: AWS_ACCESS_KEY_ID
```

Comment:
Fancy trying to set this up with an IAM role instead?

Reply:
It should just need an annotation like this: https://github.com/zooniverse/operations/blob/master/kubernetes/alb-ingress-controller.yaml#L30

Reply:
With the appropriate role name, obviously :)

Reply:
Is that for all workers or just the deployment pods? I couldn't find any docs on this from 2 seconds of looking.

Reply:
The `kube2iam` service will assign the role to the pods, so each app we deploy can have it's own role now.

Reply:
I've updated it with an IAM role...mind checking it again?